### DISCUSSION TITLE
[FR] Add possibility to borders to styles

### DISCUSSION THREAD

**Discussion by scargill** [Clooos/Bubble-Card#1125] [Feature requests]
I can't seem to get card-mod to work - I always add a consistent border around my buttons and tiles - and I can find no way to add a border around the otherwise superb climate card just installed tonight - 

Otherwise seems to work perfectly....

**Comment by J04DAN** [comment]
Add this as a custom style as a workaround

.bubble-button-card-container {
  border: solid 1px rgb(50,50,50);
  box-shadow: 0px 0px 10px -9px black
}
